python-lib/sharepoint_client.py

```diff
@@ -447,15 +469,17 @@ def __init__(self, sharepoint_user_name, sharepoint_password, sharepoint_tenant,
 
     def get(self, url, headers=None, params=None):
         headers = {} if headers is None else headers
-        headers["accept"] = DSSConstants.APPLICATION_JSON
+        headers["Accept"] = DSSConstants.APPLICATION_JSON
         headers["Authorization"] = self.get_authorization_bearer()
         return requests.get(url, headers=headers, params=params)
 
-    def post(self, url, headers=None, json=None, data=None):
-        headers = {} if headers is None else headers
-        headers["accept"] = DSSConstants.APPLICATION_JSON
-        headers["Authorization"] = self.get_authorization_bearer()
-        return requests.post(url, headers=headers, json=json, data=data)
+    def post(self, url, headers={}, json=None, data=None):
+        default_headers = {}
+        default_headers["Accept"] = DSSConstants.APPLICATION_JSON_NOMETADATA
+        default_headers["Content-Type"] = DSSConstants.APPLICATION_JSON_NOMETADATA
+        default_headers["Authorization"] = self.get_authorization_bearer()
+        default_headers.update(headers)
+        return requests.post(url, headers=default_headers, json=json, data=data)
```

Comment:
nitpick: By doing so, default_header stop being default and is now header. I would instead do the opposite
```suggestion
        headers.update(default_headers)
        return requests.post(url, headers=headers, json=json, data=data)
```

Reply:
That would overwrite what's in `headers` with the default headers though, and we want it the other way round...